docker-config/os-conductor/secrets/dev-secrets/to_env_vars.py

```diff
@@ -1,24 +0,0 @@
-import json
```

Comment:
We should keep this - might be useful in case we need to replace the environment varuables conductor is using.